src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -3481,6 +3485,17 @@ components:
         properties:
           type: object
           description: Additional JSON metadata about the dataset (this does not need to adhere to a particular schema)
+        dedicatedIngestServiceAccount:
+          type: boolean
+          default: false
+          description: >
```

Comment:
It would be useful to mention why a user may want to set this to true

Reply:
good call!